tox.ini

```
@@ -11,7 +11,13 @@ python =
     3.9: py39
 
 [testenv]
-deps = -rrequirements-dev.txt
+deps =
+    mock
+    pytest
+    pytest-cov
+    mypy
+    types-pytz
+    pytz: pytz
 commands =
     py.test test_schedule.py schedule -v --cov schedule --cov-report term-missing
     python -m mypy -p schedule
```

Add this part to fix dependency error for `types-pytz` when running `mypy` tests.

```suggestion
    python -m mypy -p schedule --install-types --non-interactive
```

---

What error are you referring to? I can't find it... 

`types-pytz` is already in the `deps` list so it should already be installed, what does `install-types` do extra?

---

Hi @SijmenHuizenga, 

When you run:
```sh
python -m mypy -p schedule
```

Then you will receive the following messages:
```text
schedule\__init__.py:489: error: Library stubs not installed for "pytz" (or incompatible with Python 3.7)
schedule\__init__.py:489: note: Hint: "python3 -m pip install types-pytz"
schedule\__init__.py:489: note: (or run "mypy --install-types" to install all missing stub packages)
schedule\__init__.py:489: note: See https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-imports
```

You can check the documentation for `--install-types` here:
https://mypy.readthedocs.io/en/stable/command_line.html#cmdoption-mypy-install-types

However, if you run:
```sh
python -m mypy -p schedule --install-types
```

You will then get the message:
```text
Installing missing stub packages:
~\schedule\.venv\Scripts\python.exe -m pip install types-pytz

Install? [yN]
```

Which is an issue because it's asking for your interaction. Which is not possible when this script is being run on the GitHub Actions level.

So, therefore, we overcome it like this:
```sh
python -m mypy -p schedule --install-types --non-interactive
```

You can check the documentation for `--non-interactive` here:
https://mypy.readthedocs.io/en/stable/command_line.html#cmdoption-mypy-non-interactive

Basically, the `types-pytz` is required for `mypy` script to do Unit Tests during the GitHub actions level. It is technically _only_ required at this stage. Meaning that it is _not_ required for the normal function of the `schedule` package. Therefore, it is _not_ advisable to add the `types-pytz` to the `requirements.txt` file.

I hope that makes sense?

---

Aaah yess! That makes perfect sense, thanks. So this error doesn't happen in Github actions is because `types-pytz` is installed before `mypy` runs (as part of dependencies).